rust/Cargo.toml

```diff
@@ -15,6 +15,7 @@ rayon = "1.7.0"
 geo = { version = "0.28.0", features = ["use-serde"] }
 geo-types = "0.7.12"
 geojson = { version = "0.24.1" }
+proj = "0.27.2"
```

Comment:
Looks like this isn't being used currently. Having this in actually causes a build failure on my end (which could probably be resolved) but thinking we should remove it for now until we actually start doing CRS projections.

Reply:
right. at the outset i was attempting to include the CRS processing that you referred to in our convo yesterday. i'll comment this out and we can come back around to this in the future, but i also had this problem on my m1 and so suspect we may want to make it an optional compass feature.